src/components/tile/__tests__/tile-test.js

```diff
@@ -0,0 +1,19 @@
+/** @jsx React.DOM */
+
+jest.dontMock('../code/views/tileView.jsx');
+
+var React = require('react/addons');
+var TileComponent = require('../code/views/tileView.jsx');
+var TestUtils = React.addons.TestUtils;
+
+describe('TileComponent', function() {
+
+  it('is an element', function() {
+    var img = {
+      src: 'foo',
+      alt: 'bar'
+    };
+    expect(TestUtils.isElement(<TileComponent image={img} title='title' />)).toBeTruthy();
```

Comment:
I completely agree  about children elements, this is something we know about and will do, as soon as we have an image component ready then we will modify this to use `this.children`